src/inverseRobot.js

```diff
@@ -24,7 +24,25 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+  const robotKeys = Object.keys(robot);
+  const robotValues = Object.values(robot);
+
+  for (let i = 0; i < robotKeys.length; i++) {
+    for (let j = i + 1; j < robotKeys.length; j++) {
+      if (robotValues[i] === robotValues[j]) {
+        return null;
+      }
+    }
+  }
```

Comment:
This nested loop is redundant, it is possible to solve this task using only one of your forEach loops.
Let'ssimplify your code!)
if you want to avoid duplicates you can just not include the already existing property in the new object. Inside your forEach callback: 
if you have this property in your new object, return null, 
and only if not, add such property to the new object.